README.md

```diff
@@ -0,0 +1,259 @@
+
+```python
+from storages.backends.s3boto import S3BotoStorage
+from require.storage import OptimizedFilesMixin
+
+# S3 storage with r.js optimization.
+class OptimizedS3BotoStorage(OptimizedFilesMixin, S3BotoStorage):
+    pass
+
+# S3 storage with r.js optimization and MD5 fingerprinting.
+from django.contrib.staticfiles.storage import CachedFilesMixin
+class OptimizedCachedS3BotoStorage(OptimizedFilesMixin, CachedFilesMixin, S3BotoStorage):
+    pass
+```
+
+For ready-made storage classes that combine django-require with Amazon S3, check out [`django-require-s3`](https://github.com/etianen/django-require-s3).
+
+Another example is when you combine `django-require` with [`django-pipeline`](https://github.com/cyberdelia/django-pipeline):
+
+Create `yourmodule/storage.py`:
+```python
+from pipeline.storage import PipelineMixin
+from require.storage import OptimizedFilesMixin
+from django.contrib.staticfiles.storage import ManifestStaticFilesStorage
+
+
+class MyStorage(
+        PipelineMixin, OptimizedFilesMixin, ManifestStaticFilesStorage):
+    pass
+```
+
+And then in your `settings.py`, use it:
+```python
+STATICFILES_STORAGE = 'yourmodule.storage.MyStorage'
+```
+
+Tests
+-----
+
+You can run the test suite from the root of the source checkout, using [tox](https://tox.readthedocs.org/en/latest/):
+
+    tox -e py35-django18
+
+For the available environments, consult `tox.ini`.
+
+Test coverage reports will be automatically generated via `tox` testruns, however if you want to see the HTML output, run `coverage html` after the tests finished.
+
+Then, open `htmlcov/index.html` in a browser to see the HTML coverage report.
+
+Support and announcements
+-------------------------
+
+Downloads and bug tracking can be found at the [main project
+website](http://github.com/etianen/django-require).
+
+You can keep up to date with the latest announcements by joining the [django-require discussion group](http://groups.google.com/group/django-require).
+
+More information
+----------------
+
+The django-require project was developed by Dave Hall. You can get the code from the [django-require project site](http://github.com/etianen/django-require).
+
+Dave Hall is a freelance web developer, based in Cambridge, UK. You can
+usually find him on the Internet in a number of different places:
+
+-  [Website](http://www.etianen.com/)
+-  [Twitter](http://twitter.com/etianen)
+-  [Google Profile](http://www.google.com/profiles/david.etianen)
+
+A V2.0.0 upgrade was done by László Károlyi. For details about him, see the following links:
+
+- [Linkedin](https://linkedin.com/in/karolyi)
+- [GitHub](https://github.com/karolyi)
+- [Google Plus](https://plus.google.com/+LaszloKAROLYI)
+
+The upgrade included:
+
+- Raising tests coverage
+- Adding `flake8` and `isort` with `tox`
+- Upgrading `.travis.yml` to use `tox`
+- Adding the options `relative_baseurl`, `entry_file_name` and `devel_tag`
+- Refactoring code parts
+- Adding various badges
+- Updating resources
```

Comment:
This file was originally written as Markdown, but changed to RST as it will then be correctly formatted on PyPi.

Do you know of a way to use Markdown and keep formatting on PyPi?